plugin/src/test/java/io/jenkins/plugins/analysis/warnings/recorder/AffectedFilesResolverITest.java

```diff
@@ -232,55 +152,46 @@ public void shouldShowFileOutsideWorkspaceIfConfigured() {
         recorder.setSourceDirectory(buildsFolder);
 
         // First build: copying the affected file is forbidden
-        TableRowPageObject unresolvedRow = buildAndVerifyFilesResolving(job,
-                "0 copied", "1 not in workspace", "0 not-found", "0 with I/O error");
+        buildAndVerifyFilesResolving(job,"0 copied", "1 not in workspace", "0 not-found", "0 with I/O error");
 
         AnalysisResult result = getAnalysisResult(job.getLastCompletedBuild());
         assertThat(result.getErrorMessages()).contains(
                 String.format("Additional source directory '%s' must be registered in Jenkins system configuration",
                         buildsFolder));
 
-        assertThat(unresolvedRow.hasLink(AFFECTED_FILE)).isFalse();
 
         WarningsPluginConfiguration.getInstance().setSourceDirectories(
                 Collections.singletonList(new SourceDirectory(buildsFolder)));
 
         // Second build: copying the affected file is permitted
-        TableRowPageObject resolvedRow = buildAndVerifyFilesResolving(job,
-                "1 copied", "0 not in workspace", "0 not-found", "0 with I/O error");
-
-        assertThat(resolvedRow.hasLink(AFFECTED_FILE)).isTrue();
-
-        SourceCodeView sourceCodeView = openSourceCode(resolvedRow);
-        assertThat(sourceCodeView.getSourceCode()).contains("<io.jenkins.plugins.analysis.core.steps.IssuesRecorder>");
-    }
-
-    private SourceCodeView openSourceCode(final TableRowPageObject resolvedRow) {
-        return new SourceCodeView(resolvedRow.clickColumnLink(AFFECTED_FILE));
+        buildAndVerifyFilesResolving(job,"1 copied", "0 not in workspace", "0 not-found", "0 with I/O error");
     }
 
-    private TableRowPageObject buildAndVerifyFilesResolving(final FreeStyleProject job, final String... resolveMessages) {
+    private void buildAndVerifyFilesResolving(final FreeStyleProject job,
+            final String... resolveMessages) {
         AnalysisResult result = scheduleBuildAndAssertStatus(job, Result.SUCCESS);
 
         assertThat(getConsoleLog(result)).contains(resolveMessages);
 
-        TablePageObject issues = getIssuesTable(result);
-        assertThat(issues.getColumnHeaders()).containsExactly(
-                DETAILS, AFFECTED_FILE, SEVERITY, AGE);
-        List<TableRowPageObject> rows = issues.getRows();
-        assertThat(rows).hasSize(1);
-        TableRowPageObject row = rows.get(0);
-        assertThat(row.getValuesByColumnLabel()).contains(
-                entry(DETAILS, "foo defined but not used"),
-                entry(AFFECTED_FILE, "config.xml:451"),
-                entry(SEVERITY, "Normal"));
-        return row;
+        assertThat(result.getIssues()).hasSize(1);
+
+        IssuesRow firstRow = getIssuesModel(result, 0);
+        assertThat(firstRow.getSeverity()).contains(Severity.WARNING_NORMAL.getName());
+        assertThat(firstRow.getFileName().getDisplay()).contains("config.xml:451");
```

Comment:
Hier noch das `<a>` link prüfen

Reply:
2 der 3 Tests, die die Methode aufrufen, um die Assertion durchzuführen, haben an dieser Stelle keinen Link. Vermutlich wurde es vorher deshalb auch nicht geprüft, kann das sein?

Reply:
Das habe ich wohl übersehen. Wenn das so ist, wäre vielleicht ein Boolean Parameter sinnvoll, der dann entscheidet, ob mit oder ohne `<a>`?

Reply:
Dafür steck ich leider nicht tief genug drin, wie das Frontend des Jenkins zusammengebaut wird zur Laufzeit. Ich war schon erstaunt, dass in dem Java-Objekt ein HTML-String drin steckt....wirkt für mich so, als würde das Backend bereits Frontend-Code generieren?

Reply:
Genau, einige statische Analyse Tools erzeugen auch HTML messages.

Reply:
Ich dachte einfach folgenden Check (siehe alter Kommentar). 
```
if (isFileNameClickable) {
    assertThat(row.getFilename()).contains("<a href=");
} 
else {
 assertThat(row.getFilename()).doesNotContain("<a href=");
}
```

Reply:
Jetzt versteh ich glaub ich was Sie meinen. Den Boolean lediglich im Test halten; nicht als Teil des Objektes, das von Jenkins im "Normalbetrieb" verarbeitet wird. Das lässt sich natürlich ohne großen Aufwand umsetzen.